src/super_gradients/recipes/dataset_params/coco_detection_yolox_dataset_params.yaml

```diff
@@ -5,19 +5,29 @@ train_json_file: instances_train2017.json # path to coco train json file, data_d
 val_json_file: instances_val2017.json # path to coco validation json file, data_dir/annotations/val_json_file.
 
 batch_size: 16 # batch size for trainset in CoCoDetectionDatasetInterface
-val_batch_size: 128 # batch size for valset in CoCoDetectionDatasetInterface
+val_batch_size: 64 # batch size for valset in CoCoDetectionDatasetInterface
 val_image_size: 640 # image size for valset in CoCoDetectionDatasetInterface
 train_image_size: 640 # image size for trainset in CoCoDetectionDatasetInterface
 
-mixup_prob: 1.0
-degrees: 10.
-shear: 2.0
-flip_prob: 0.5
-hsv_prob: 1.0
-mosaic_scale: [0.1, 2]
-mixup_scale: [0.5, 1.5]
-mosaic_prob: 1.
-translate: 0.1
+# Augmentations
+mixup_prob: 1.0 # probability to apply per-sample mixup
+degrees: 10. # rotation degrees, randomly sampled from [-degrees, degrees]
+shear: 2.0 # shear degrees, randomly sampled from [-degrees, degrees]
```

Comment:
I don't think shear is measured in degrees, is it?

Reply:
nope.